src/main/java/org/alliancegenome/curation_api/jobs/processors/BulkLoadProcessor.java

```diff
@@ -136,6 +136,7 @@ protected void processFilePath(BulkLoad bulkLoad, String localFilePath, Boolean
 			bulkLoadFile = bulkLoadFiles.getResults().get(0);
 			if (bulkLoadFile.getBulkloadStatus().isNotRunning()) {
 				bulkLoadFile.setLocalFilePath(localFilePath);
+				bulkLoadFile.setErrorMessage(null);				
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace